src/server/scripts/Outland/zone_hellfire_peninsula.cpp

```diff
@@ -1039,6 +1041,212 @@ class npc_magister_aledis : public CreatureScript
     }
 };
 
+enum WatchCommanderLeonus
+{
+    SAY_COVER                   = 0,
+    EVENT_LEONUS_TALK           = 1,
+    EVENT_INFERNAL_RAIN_ATTACK  = 2,
+    EVENT_FEAR_CONTROLLER_CAST  = 3,
+    EVENT_ACTIVE_FALSE          = 4,
+    NPC_INFERNAL_RAIN           = 18729,
+    SPELL_INFERNAL_RAIN         = 33814,
+    NPC_FEAR_CONTROLLER         = 19393,
+    DATA_ACTIVE                 = 1,
+};
+
+struct npc_watch_commander_leonus : public ScriptedAI
+{
+    npc_watch_commander_leonus(Creature* creature) : ScriptedAI(creature) { }
+
+    void Reset() override
+    {
```

Comment:
_events.Reset() missing here

Reply:
Added in the line below your comment, so github can't hide your comment.